i6_models/parts/conformer/mhsa.py

```diff
@@ -2,10 +2,10 @@
 
 __all__ = ["ConformerMHSAV1", "ConformerMHSAV1Config"]
 from dataclasses import dataclass
-from typing import Optional
 import torch
 
 from i6_models.config import ModelConfiguration
+from i6_models.util.compat import logical_not
```

Comment:
I would prefer to just import the module, like:
```suggestion
from i6_models.util import compat
```
And then below use `compat.logical_not`.